encode/src/main/scala/org/broadinstitute/monster/etl/encode/extract/EncodeIngestExtract.scala

```diff
@@ -0,0 +1,148 @@
+package org.broadinstitute.monster.etl.encode.extract
+
+import caseapp.HelpMessage
+import com.spotify.scio.ContextAndArgs
+import org.broadinstitute.monster.etl.encode.EncodeEntity
+import org.broadinstitute.monster.etl.encode.extract.client.EncodeClient
+import cats.effect.{ExitCode, IO, IOApp}
+import cats.implicits._
+import com.spotify.scio.extra.json._
+import com.spotify.scio.values.SCollection
+import io.circe.JsonObject
+import io.circe.syntax._
+
+object EncodeIngestExtract extends IOApp {
+
+  case class Args(
+    @HelpMessage("List of Assays")
+    assayTypes: List[String],
+    @HelpMessage("Output dir to JSON for ENCODE outputs")
+    outputDir: String
+  )
+
+  override def run(rawArgs: List[String]): IO[ExitCode] = {
+    val (pipelineContext, parsedArgs) = ContextAndArgs.typed[Args](rawArgs.toArray)
+
+    EncodeClient.resource.use { client =>
+      // for each entity, get the params and the extract based off of those params
+      val encodeExtractions = new EncodeExtractions(client)
+
+      // make assayTypes in scollection
+      val experimentsSearchParams = encodeExtractions.getExperimentSearchParams(
```

Comment:
Here an elsewhere, can you add newlines between the `val` definitions and the `saveAsJsonFile` call? Makes it easier to read.